docs/local-setup.md

```diff
@@ -53,31 +56,40 @@ cp .env.example .env
 
 ---
 
-#### 2. Install Composer Dependencies
+#### 2. Install Dependencies
+
+##### Composer Dependencies
 
 ```bash
-docker run --rm \
-    -u "$(id -u):$(id -g)" \
-    -v "$(pwd):/var/www/html" \
-    -w /var/www/html \
-    laravelsail/php82-composer:latest \
-    composer install --ignore-platform-reqs --no-scripts
+pls ih composer install --ignore-platform-reqs
+```
+
+##### JS Dependencies Installation and Build
+
+```bash
+pls ih npm ci && npm run build
 ```
 
 ---
 
 #### 3. Start the containers and open a shell into the main PHP container
 
+Generate an encryption key:
```

Comment:
This is instructed outside of the shell, but also mentioned below `Generate an App Key:`

Reply:
Fixed